tests/test_upgrade_sr_v2.py

```diff
@@ -355,70 +384,103 @@ def test_vote(
             CSM_AFTER_VOTE["minDepositBlockDistance"],
         ),
     )
-    validate_grant_role_event(events[20], REQUEST_BURN_SHARES_ROLE, CS_ACCOUNTING_ADDRESS, contracts.agent.address)
-    validate_grant_role_event(events[21], RESUME_ROLE, contracts.agent.address, contracts.agent.address)
+    validate_grant_role_event(events[20], REQUEST_BURN_SHARES_ROLE, CS_ACCOUNTING_ADDRESS, AGENT)
+    validate_grant_role_event(events[21], RESUME_ROLE, AGENT, AGENT)
 
     validate_resume_event(events[22])
-    validate_revoke_role_event(events[23], RESUME_ROLE, contracts.agent.address, contracts.agent.address)
+    validate_revoke_role_event(events[23], RESUME_ROLE, AGENT, AGENT)
     validate_updateInitial_epoch(events[24])
 
     validate_evmscript_factory_added_event(
         events[25],
         EVMScriptFactoryAdded(
             factory_addr=EASYTRACK_CSM_SETTLE_EL_REWARDS_STEALING_PENALTY_FACTORY,
-            permissions=(create_permissions(contracts.csm, "settleELRewardsStealingPenalty")),
+            permissions=(create_permissions(csm, "settleELRewardsStealingPenalty")),
         ),
         ["LogScriptCall", "EVMScriptFactoryAdded", "ScriptResult", "ExecuteVote"],
     )
 
 
+def get_staking_router():
+    return interface.StakingRouter(STAKING_ROUTER)
+
+
+def get_deposit_security_module_v3():
+    return interface.DepositSecurityModule(DEPOSIT_SECURITY_MODULE_V3)
+
+
+def get_deposit_security_module_v2():
+    return interface.DepositSecurityModule(DEPOSIT_SECURITY_MODULE_V2)
+
+
+def get_vebo():
+    return interface.ValidatorsExitBusOracle(VALIDATORS_EXIT_BUS_ORACLE)
+
+
+def get_simple_dvt():
+    return interface.SimpleDVT(SIMPLE_DVT)
+
+
+def get_burner():
+    return interface.Burner(BURNER)
+
+
+def get_csm():
+    return interface.CSModule(CSM_ADDRESS)
+
+
+def get_voting():
+    return interface.Voting(VOTING)
+
+
+def get_csm_hash_consensus():
+    return interface.CSHashConsensus(CS_ORACLE_HASH_CONSENSUS_ADDRESS)
+
+
 def check_ossifiable_proxy_impl(proxy, expected_impl):
     current_impl_address = proxy.proxy__getImplementation()
     assert current_impl_address == expected_impl, f"Expected {expected_impl} impl but got {current_impl_address}"
 
 
 def check_dsm_roles_before_vote():
-    new_dsm_doesnt_have_unvet_role = contracts.staking_router.hasRole(
-        STAKING_MODULE_UNVETTING_ROLE, contracts.deposit_security_module
-    )
+    staking_router = get_staking_router()
+    deposit_security_module_v2 = get_deposit_security_module_v2()
+    deposit_security_module_v3 = get_deposit_security_module_v3()
+    new_dsm_doesnt_have_unvet_role = staking_router.hasRole(STAKING_MODULE_UNVETTING_ROLE, deposit_security_module_v3)
     assert not new_dsm_doesnt_have_unvet_role
```

Comment:
The variable contains the result of the method `hasRole` call, but the name claims otherwise

```suggestion
    new_dsm_has_unvet_role = staking_router.hasRole(STAKING_MODULE_UNVETTING_ROLE, deposit_security_module_v3)
    assert not new_dsm_has_unvet_role
```